binary_search/bin_search.cpp

```diff
@@ -0,0 +1,38 @@
+#include<bits/stdc++.h>
+#include<algorithm>
+using namespace std;
+
+int a[10]={7,10,22,30,33,45,66,72,83,100};
+int binsearch(int st,int en,int el)
+  {
```

Comment:
Incorrect spacing. Please try to adhere to best practices of the particular language.